website/docs/tutorials/08-interacting-with-starknet/01-invoke.md

```diff
@@ -0,0 +1,37 @@
+# Invoking contracts
+
+## Overview
+This cheatcode allows you to send an invoke transaction with `@external` function entrypoint.
+
+The basic inputs that you need for the commands are:
+- [Signing credentials](#signing)
+- Contract address
+- Function name
+- Inputs to the function
+- Fee specification (concrete amount, or auto-estimation)
+- Network you want to target (i.e. its name or gateway URL)
+
+For detailed API description, see [invoke command reference](../../cli-reference.md#invoke).
+
+## Signing
+Credentials for paying the fee are needed, which are: 
+- Account address (which pays the fee)
+- Private key for that account (from ArgentX, Braavos etc.) - in hex string (i.e. 0xa1c359ef) form. This can be provided with `PROTOSTAR_ACCOUNT_PRIVATE_KEY` env variable or with a file on local filesystem containing that key in plaintext, in which case you should use `--private-key-path`.
+
+Custom signing logic is made possible by using custom signers - see details [here](../06-deploying/01-cli.md/#2-using-a-custom-signer-class).
+
+## Usage example
+
+```shell 
+PROTOSTAR_ACCOUNT_PRIVATE_KEY=0x<redacted> protostar invoke --contract-address 2104712876024647862991828835908656456713967333335715277181131715044727499025 --function "setter_tester_success" --network testnet --account-address 0x0481Eed2e02b1ff19Fd32429801f28a59DEa630d81189E39c80F2F60139b381a --max-fee auto --inputs 3
+16:54:56 [INFO] Invoke transaction was sent.
+Transaction hash: 0x05d2362b9b5a5aba8a02a41d2f1fcbdc06cde89f90cf33c0ea4957846c86aeef
+16:54:57 [INFO] Execution time: 13.17 s
+```
+:::warning
+Setting `max-fee` to `auto` is discouraged, since it may incur extra unexpected costs.
```

Comment:
It's discouraged when a real account is used. I'd include the link to the Starknet.py docs.

Reply:
How is starknet.py relevant here?
I think that explanation is good enough, because:
- If i include the info to watch out for your funds with a "real" account, then i have to make the distinction between accounts on testnet/mainnet, those that have actual ETH bridged to them etc. I think that best we can do here to prevent accidents is to tell the user that _"Setting `max-fee` to `auto` is discouraged (especially on mainnet)"_.
- If i link to starknet.py i create a non-obvious connection (for the user) between the SDK docs and our docs, which should be covered by us.